english/marshmallow.md

```diff
@@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
 
 Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
 
-Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
```

Comment:
Hi Caroline,

Here’s the link to the new commit:

https://github.com/bynumjj/create-your-own-adventure/commit/04ac4a1dc7aeeb836c16eb960ebbcb24274cc667 <https://github.com/bynumjj/create-your-own-adventure/commit/04ac4a1dc7aeeb836c16eb960ebbcb24274cc667>

Per the instructions from one of the Udacity tutors, I merged it and pushed it to my master branch.

Thanks for taking a look.  Let me know if I need to do anything else.

Best,

Jeff

> On Nov 5, 2015, at 11:27 AM, Caroline Buckey <notifications@github.com> wrote:
> 
> In english/marshmallow.md <https://github.com/udacity/create-your-own-adventure/pull/2685#discussion_r44056797>:
> 
> > @@ -115,7 +115,7 @@ Something lightly brushes the back of your neck, sending a chill down your spine
> >  
> >  Startled, you immediately whirl around to see who (or what) touched you...but there's no one behind you.
> >  
> > -Click [here](./sonic-screwdriver/sonic-screwdriver.md) to use your sonic screwdriver to scan the air 
> It shouldn't be necessary to create a new pull request, but I don't see the change. Make sure you've pushed to the prostate branch in your GitHub repository.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/udacity/create-your-own-adventure/pull/2685/files#r44056797>.
>